### PR TITLE
[web-service] fix parsing of ipv6 addresses for status page

### PR DESCRIPTION
**Pull request by martinzi**
fixed parsing of ipv6 addresses + removed duplicate state and add txpower

### CODE REVIEW

**Comment by bukepo** [comment]
@MarekPorwisz Could you sign the CLA?

**Comment by MarekPorwisz** [comment]
> @MarekPorwisz Could you sign the CLA?

Me? I think you meant @martinzi

**Comment by bukepo** [comment]
@martinzi could you sign the CLA?

**Comment by martinzi** [comment]
@bukepo CLA approval process is in its final steps, might take a couple more days to complete.

**Comment by martinzi** [comment]
@googlebot I signed it!

**Comment by martinzi** [comment]
#685

**Comment by martinzi** [comment]
@googlebot I signed it!
Can you find it?

File: src/web/web-service/wpan_service.cpp [hunk 1]
```diff
@@ -259,16 +259,22 @@ std::string WpanService::HandleStatusRequest()
     }
 
     VerifyOrExit((rval = client.Execute("version")) != nullptr, ret = kWpanStatus_GetPropertyFailed);
-    networkInfo["NCP:Version"] = rval;
+    networkInfo["OT-CTL:Version"] = rval;
+
+    VerifyOrExit((rval = client.Execute("version api")) != nullptr, ret = kWpanStatus_GetPropertyFailed);
+    networkInfo["OT-CTL:Version API"] = rval;
```
Comment (bukepo) [inline src/web/web-service/wpan_service.cpp:265]:
```suggestion
    networkInfo["OpenThread:Version"] = rval;

    VerifyOrExit((rval = client.Execute("version api")) != nullptr, ret = kWpanStatus_GetPropertyFailed);
    networkInfo["OpenThread:Version API"] = rval;
```

File: src/web/web-service/wpan_service.cpp [hunk 2]
```diff
@@ -309,17 +327,34 @@ std::string WpanService::HandleStatusRequest()
...
+                    continue;
+                }
+
+                if (strstr(rval, localAddressToken) != rval)
+                {
+                    networkInfo["IPv6:GlobalAddress"] = rval;
+                }
+                else
+                {
+                    networkInfo["IPv6:LocalAddress"] = rval;
```
Comment (bukepo) [inline src/web/web-service/wpan_service.cpp:342]:
```suggestion
                    networkInfo["IPv6:LinkLocalAddress"] = rval;
```

File: src/web/web-service/wpan_service.cpp [hunk 3]
```diff
@@ -279,28 +285,40 @@ std::string WpanService::HandleStatusRequest()
...
             char *meshLocalAddressToken = nullptr;
 
-            if (strstr(rval, meshLocalPrefix.c_str()) != rval)
+            if (strstr(rval, "> ") != nullptr)
+            {
+                rval += 2;
+            }
+
+            if (strstr(rval, linkLocalAddressToken) == rval)
             {
```
Comment (martinzi) [inline src/web/web-service/wpan_service.cpp:322]:
```suggestion
            {
                networkInfo["IPv6:LinkLocalAddress"] = rval;
```

**Review by martinzi — COMMENTED** [review]
Your suggestions are added, thx.

**Review by bukepo — APPROVED** [review]
LGTM:+1:

**Review by jwhui — APPROVED** [review]

**Pull request merged**